spec/gutentag/active_record_spec.rb

```diff
@@ -41,6 +42,14 @@
       it { is_expected.not_to include oregon_article }
     end
 
+    context 'given a denormalized tag name' do
```

Comment:
that's something best done on master or in a separate PR ... otherwise this will be a big/unreadable diff and cause merge conflicts ...